package.json

```diff
@@ -25,18 +25,12 @@
   },
   "devDependencies": {
     "cross-env": "^7.0.3",
-    "husky": "^7.0.4",
     "lerna": "^3.22.1",
     "prettier": "2.2.1",
     "pretty-quick": "^3.1.0",
     "rimraf": "^3.0.2",
     "typescript": "^4.1.3"
   },
-  "husky": {
-    "hooks": {
-      "pre-push": "yarn test && pretty-quick --staged"
```

Comment:
Hmm maybe; but we still don't autoformat in CI so it doesn't matter until we do that. I plan to add an autoformat job in CI this week if possible.

Reply:
Doesn't this have to be pre-commit tho? Not sure why CI needs to format/lint things. Will it push another commit if it fixes anything?

Reply:
CI needs to check that it _is_ formatted; I agree that in order to actually get things from unformatted to formatted we need to do it locally, but I also think it's better to do that in-editor rather than pre-commit.

Reply:
But we need CI in order to ensure that once we autoformat our codebase, it stays formatted.

Reply:
Both make sense to me. Let's go ahead with this then.